code/go/0chain.net/blobbercore/blobberhttp/response.go

```diff
@@ -34,6 +34,14 @@ type ReferencePathResult struct {
 	LatestWM *writemarker.WriteMarker `json:"latest_write_marker"`
 }
 
+type RefResult struct {
+	TotalPages    int                      `json:"total_pages"`
+	NewOffsetPath string                   `json:"offsetPath,omitempty"`
```

Comment:
could you add some documentation about what is NewOffsetPath/NewoffsetDate?  it is confused there without any comment.

Reply:
`total_pages` is snake_case, and offsetPath is lower_case. should we use snake_case only?

Reply:
and the field name is NewOffsetPath, but it is `offsetPath` in json. why they are different?